src/DateInput.tsx

```diff
@@ -358,3 +360,84 @@ export function DateInputFieldWithLabel (
     </>
   )
 }
+
+// Type-in Date Input
+
+const dateMask = [/\d/, /\d/, '/', /\d/, /\d/, '/', /\d/, /\d/, /\d/, /\d/]
+
+export function DateInputTypeIn ({
+  name,
+  hasError,
+  onChange,
+  onBlur
+}: {
+  name: string
+  hasError?: boolean
+  onChange(val: string): void
+  onBlur(): void
+}) {
+  return (
+    <MaskedInput
+      name={name}
+      mask={dateMask}
+      id='date'
+      placeholder='mm/dd/yyyy'
+      type='text'
+      onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+        onChange(e.target.value)
+      }}
+      onBlur={onBlur}
+      render={(inputRef, props) => (
+        <Input
+          name={name}
+          ref={inputRef}
+          hasError={hasError}
+          onChange={onChange}
+          onBlur={onBlur}
```

Comment:
Do we really want `onChange` and `onBlur` passed in two places, or does `{...props}` override what's passed to `Input` anyway?